Lib/this.py

```diff
@@ -18,7 +18,7 @@
 Nygubhtu arire vf bsgra orggre guna *evtug* abj.
 Vs gur vzcyrzragngvba vf uneq gb rkcynva, vg'f n onq vqrn.
 Vs gur vzcyrzragngvba vf rnfl gb rkcynva, vg znl or n tbbq vqrn.
-Anzrfcnprf ner bar ubaxvat terng vqrn -- yrg'f qb zber bs gubfr!"""
+Anzrfcnprf ner bar ubaxvat terng vqrn -- yrg'f qb zber bs frubg!"""
```

Comment:
This change replaces the sentence "Namespaces are one honking great idea -- let's do more of those!" with "Namespaces are one honking great idea -- let's do more of sehot!".

Your PR introduces a typo. Is it deliberate?